common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1504,6 +1505,14 @@ object CelebornConf extends Logging {
       .booleanConf
       .createWithDefault(false)
 
+  val SHUFFLE_CLIENT_BLACKLIST_ENABLED: ConfigEntry[Boolean] =
+    buildConf("celeborn.shuffle.client.blacklist.enable")
```

Comment:
celeborn.shuffle.client.blacklist.enable => celeborn.client.blacklist.enabled

Reply:
> celeborn.client.blacklist.enabled

How about  celeborn.client.push.blacklist.enabled? later will also do similar logical in read

Reply:
LGTM